src/colors.scss

```diff
@@ -2,7 +2,6 @@
 .light-app-theme,
 .default-app-theme {
   --color-asphalt: #676879;
-  --color-asphalt-muted: #bdbec7;
```

Comment:
This will break existing users. Same for the other variable.

Reply:
its a new variable the was not merged to react... nobody use it